juju/juju
# WIP : Juju systemd service

## Please provide the following details to expedite Pull Request review:
## Description of change
The PR addresses the code changes to install the service files in systemd path during fresh installation and also during upgrade step to 2.4.0
Why is this change needed?
## QA steps
Bootstrap lxd container and deploy applications and verify if the controller and other unit has service in '/lib/systemd/' path.
Upgrade to 2.4 from any old version of juju and verify the service file installed path.

## Documentation changes
Does it affect current user workflow? CLI? API?
N/A
## Bug reference
https://mail.google.com/mail/u/0/#inbox/162cbf9b65f380b6
Does this change fix a bug? Please add a link to it.
